.github/workflows/wcag_test.yml

```diff
@@ -6,11 +6,11 @@ on:
 
 jobs:
   WCAG-test:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
for my own understanding, why is this change needed?

Reply:
I was consistently running into this error when running the wcag tests via CI, but couldn't reproduce locally
https://github.com/yext/answers-search-ui/runs/5606193735?check_suite_focus=true
from looking things up online it seems to be a lower level error that people don't really know the solution for, so I started off by seeing if making the environment closer to my local one, and for whatever reason it seems to be working now